package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "conseiljs",
-  "version": "0.1.25",
+  "version": "0.1.29-z",
```

Comment:
Since we are now going to publish only the production ConseilJS package from `master`, we should revert this to the last official version.

For staging, we should publish a new NPM package called 'ConseilJS-staging` and publish to it from a `staging` branch.

Reply:
0.1.30 for now, will discuss in person